README.md

```diff
@@ -15,13 +15,17 @@ Installation
 Usage Examples
 --------------
 
-Small usage examples have been included in the [examples/](examples/) folder. The examples are PHP scripts which render the
-templates and their atomic partials and layouts from the folders. In the PHP files you can find the most basic implementation
-example - and you can execute the examples by running them through your HTTPD or calling `php examples/example_variables.php` etc.
-
-> Tip: you can execute all examples in the same run by calling
->
-> `find examples/ -maxdepth 1 -name \*.php -exec php {} \;`
+The library source comes with a set of example scripts to study and play around with.
+They are PHP entry scripts which render templates, their partials and layouts.
+
+```bash
+$ git clone git@github.com:TYPO3/Fluid.git
+$ composer update
```

Comment:
```suggestion
$ composer install
```

Reply:
No. Update is fine since there is no composer.lock. Recent composer versions even warn about this, using 'install' when there is no .lock.

Reply:
As discussed personally I see `install` as the safer option in all cases but given that we have a library package here always updating all deps to their latest version should be OK.